modules/common/const.go

```diff
@@ -20,6 +20,8 @@ package common
 const (
 	// AppSelector - used by operators to specify labels
 	AppSelector = "service"
+	// ComponentSelector - used by operators to specify labels for a sub component
+	ComponentSelector = "component"
```

Comment:
How this relates to the VerifyNetworkStatusFromAnnotation change?

Reply:
its not. I can split that out.

Reply:
https://github.com/openstack-k8s-operators/lib-common/pull/226